module/account/tests/Domain/ValueObject/EmailTest.php

```diff
@@ -26,6 +26,7 @@ public function testValidaValue(string $value): void
         $email = new Email($value);
         $this->assertEquals($value, $email->getValue());
         $this->assertEquals($value, (string) $email);
+        $this->assertTrue($email->isEqual($email));
```

Comment:
This Assertion have no sense in this test "testValidaValue". 
Tests the methods on its own, what about different objects of the same value or with different values ?
This should be on separate test.